### PR TITLE
[onert/core] Update traininfo_loader to support loss reduction type

### PR DESCRIPTION
**Pull request by zetwhite**
This PR updates traininfo_loader to support loss reduction type.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

to-resolve : https://github.com/Samsung/ONE/issues/12606

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

File: runtime/onert/core/src/loader/traininfo_loader.cc [hunk 1]
```diff
@@ -62,6 +62,7 @@ ir::train::LossInfo loadLossInfo(const circle::ModelTraining *circle_model)
   // fill ir_loss from circle_loss
   ir::train::LossInfo ir_loss;
   const circle::LossFn circle_loss = circle_model->lossfn();
+  const circle::LossReductionType circle_loss_rdt = circle_model->loss_reduction_type();
```
Comment (zetwhite) [inline runtime/onert/core/src/loader/traininfo_loader.cc:65]: Q: why this change happened? 

From this PR - https://github.com/Samsung/ONE/pull/12607, train info schema updated to support loss reduction type.
So, Let's update the loader to support loss reduction type.

**Review by ragmani — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by Aeren1564 — APPROVED** [review]
LGTM!
